CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 2.3.1
+* Fix bad phase seek when starting from preroll. [lp1930143](https://bugs.launchpad.net/mixxx/+bug/1930143)  [#4093](https://github.com/mixxxdj/mixxx/pull/4093)
+* AutoDJCratesDAO: Fix crash after loading/unloading tracks. [lp1936396](https://bugs.launchpad.net/mixxx/ [#4107](https://github.com/mixxxdj/mixxx/pull/4107)+bug/1936396)
+
 ## [2.3.0](https://launchpad.net/mixxx/+milestone/2.3.0)
```

Comment:
Can you add the release date for the previous version? I think we should add it whenever we are going to release the version after it.

Reply:
done